### DISCUSSION TITLE
translations (i18n l10n) for tests

### DISCUSSION THREAD

**Discussion by mraesener-aubex** [robocorp/robocorp#405] [Q&A]
Hey everybody,

I was running tests for `robocorp.windows` on my machine, which is set to german ("de_DE"), and noticed that some are failing because only the english terms are expected.

I'd assume that it would be beneficial to support multiple languages than just english. if you're open for an PR, how would you approach this?

**Comment by fabioz** [comment]
I'd probably just check for the other expected results on the other languages (but it may be hard to keep it up in practice as we just have Github Actions for the CI, so, it's possible that other expected languages end up being broken without us knowing about it -- given that, I'm not sure it's worth to support it given the high chance that it'll break going forward).

Off course, actual bugs in `robocorp.windows` due to different locales should be fixed (but if this is only about the expected values I'm not sure).

**Comment by mraesener-aubex** [comment]
ok, sure, should be covered as well in CI, makes sense. 

I noticed that `close_window` has a hardcoded default locator for english language (https://github.com/robocorp/robocorp/blob/b369f2b0108422012d3e4968e39c611cb341eec8/windows/src/robocorp/windows/_window_element.py#L353)

I'd expect here a more robust default behavior, if I'm not wrong
